keyboards/mehkee96/keymaps/johann/keymap.c

```diff
@@ -0,0 +1,84 @@
+|____________|________|________|________|________|________|________|________|________|________|________|________|________|_____________|________|________|________|________|
+| BACKLIGHT    |        |        |        |        |        |        |        |        |        |        |        |                    |        |        |        |        |
+| TOGGLE       |        |        |        |        |        |        |        |        |        |        |        |                    |        |        |        |        |
+|______________|________|________|________|________|________|________|________|________|________|________|________|____________________|________|________|________|________|
+|         |        |        |        |BACKLHT |BACKLHT |BACKLHT |        |        |        |        |        |                |        |        |        |        |        |
+|         |        |        |        | DCRSE  |TOGGLE  |INCREASE|        |        |        |        |        |                |        |        |        |        |        |
+|_________|________|________|________|________|________|________|________|________|________|________|________|________________|________|________|________|________|________|
+|          |          |          |                                                        |        |        |        |        |        |        |        |        |        |
+|          |          |          |                                                        |        |        |        |        |        |        |        |        |        |
+|__________|__________|__________|________________________________________________________|________|________|________|________|________|________|________|________|________|
+BL_TOGG, BL_DEC, BL_INC changes the in-switch LEDs
+*/
+
+
+	LAYOUT(
+    RESET,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_MUTE, KC_VOLD, KC_VOLU, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______,
+    _______, RGB_TOG, _______, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, _______, _______, _______, _______,          _______, _______, _______, _______,
+    BL_TOGG, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                   _______, _______, _______, _______,
+    _______         , _______, _______, BL_DEC,  BL_TOGG, BL_INC,  _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______,
+    _______, _______, _______,                            _______,                            _______, _______, _______, _______, _______, _______, _______, _______, _______),
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {
```

Comment:
Could you remove the `action_get_macro` function?  It's been deprecated in favor of `process_record_user`.